lib/KalturaModel.php

```diff
@@ -66,6 +66,21 @@ public function getClientSideSession( $privileges = '', $expiry = 86400 ) {
 		return sanitize_text_field( $this->createKS( $this->_partnerId, $this->_userId, Kaltura_Client_Enum_SessionType::USER, $privileges, $expiry ) );
 	}
 
+	public function getAdminSessionUsingApi($partnerId, $adminSecret, $privileges = "", $expiry = 86400)
```

Comment:
The original signature of those methods was `getAdminSessionUsingApi($privileges = "", $expiry = 86400)`
Is there a need to pass `partnerId` and `adminSecret`?

See
https://github.com/kaltura/all-in-one-video-pack.wordpress/blob/2a51cef383f1fa02aecefa1b9d0fc2b7adbc7d0f/lib/KalturaModel.php#L58